openAPI/post.openAPI.json

```diff
@@ -83,6 +83,39 @@
         }
       }
     },
+    "/posts/homepage/{userid}": {
+      "get": {
+        "tags": ["posts"],
```

Comment:
I guess this should be ['users']

Reply:
Yep! we have updated it and added a new user's route. the swagger file was updated as well.